src/components/Board/Board.js

```diff
@@ -1,13 +1,30 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import Grid from '@material-ui/core/Grid';
 import Column from '../Column/Column';
+import * as cardsApi from '../../utils/cardsApi';
 
-const Board = () => (
-  <div>
-    <Grid container spacing={3}>
-      <Grid item xs={3}><Column /></Grid>
-    </Grid>
-  </div>
-);
+const Board = () => {
+  const [cards, setCards] = useState([]);
+
+  const addCard = (newCard) => {
+    setCards([...cards, newCard]);
+  };
+
+  useEffect(() => {
+    cardsApi.getCards().then((cardsResponse) => {
+      setCards(cardsResponse);
+    });
+  }, []);
```

Comment:
Great! Whenever the page is refreshed we do a fetch to our API for the cards (like componentDidMount).